06_performance_monitor/performance_monitor.rb

```diff
@@ -0,0 +1,10 @@
+def measure(n=1)
+    avg_time=0
+
+    n.times do
+        start_time=Time.now
+        yield
+        avg_time+=Time.now-start_time
+    end
+    return avg_time/n
```

Comment:
Don't need the `return` keyword`.

Also, not related to this branch only, every language has its own style guide. Learn more about Ruby Style Guide here https://rubystyle.guide/